packages/contracts/contracts/Migrations.sol

```diff
@@ -1,10 +1,11 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.19;
 
 
 contract Migrations {
   address public owner;
-  uint public last_completed_migration;
+  uint public lastCompletedMigration;
```

Comment:
I changed this variable from camel case to underscores to match the Migrations.sol file found in the `installed_contracts` packages. Migrations weren't working properly when the files didn't match.